x/decentralized-vault/client/cli/tx.go

```diff
@@ -0,0 +1,195 @@
+			msg := types.NewMsgNftLocked(clientCtx.GetFromAddress(), receiver, args[1], args[2], args[3])
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdNftUnlocked() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "nft-unlocked [target_address] [nft-id]",
+		Short: "unlocked nft on other network",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`unlocked nft on other network.
+Example:
+$ %s tx %s nft-locked ununifi1wgjh88unam4tuln0ju6l6q6cd08zk2vs87uytv a10 --from myKeyName --chain-id ununifi-x
+`, version.AppName, types.ModuleName)),
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			target, err := sdk.AccAddressFromBech32(args[0])
+			msg := types.NewMsgNftUnlocked(clientCtx.GetFromAddress(), target, args[1])
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdNftTransferRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "transfer-request [nft-id] [eth-address]",
+		Short: "nft transfer request",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`nft transfer request.
+Example:
+$ %s tx %s transfer-request a10 --from myKeyName --chain-id ununifi-x
+`, version.AppName, types.ModuleName)),
+		Args: cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgNftTransferRequest(clientCtx.GetFromAddress(), args[0], args[1])
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdRejectTransferRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "reject-transfer [nft-id]",
+		Short: "reject nft transfer request",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`reject nft transfer request.
+Example:
+$ %s tx %s reject-transfer a10 --from myKeyName --chain-id ununifi-x
+`, version.AppName, types.ModuleName)),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgNftRejectTransfer(clientCtx.GetFromAddress(), args[0])
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), &msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdTransferred() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "transferred [nft-id]",
+		Short: "nft transferred",
```

Comment:
Is this nft transfer history? Would be nice to put some more information for clarity.